.gitlab-ci.yml

```diff
@@ -169,7 +167,7 @@ before_script:
     - echo "Running windows source tests"
   tags: ["runner:windows-docker", "windowsversion:1809"]
   script:
-    - docker run --rm -m 4096M -v "$(Get-Location):c:\mnt" -e IS_AWS_CONTAINER=true -e SIGN_WINDOWS=true 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/windows_${ARCH}:$Env:DATADOG_AGENT_WINBUILDIMAGES \unittests.bat
+    - docker run --rm -m 4096M -v "$(Get-Location):c:\mnt" -e IS_AWS_CONTAINER=true -e SIGN_WINDOWS=true 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/windows_${ARCH}:$Env:DATADOG_AGENT_BUILDERS \unittests.bat
```

Comment:
the windows builders generally have a different tag; I don't think you want this change

Reply:
Ok, let's sync on this and how I could upgrade the python used in these images then (a new dep must be installed with pip). I've slightly discussed this with Kylian and Albert about the images tag but I need more infos.

Reply:
I'm in favor of using the same tag for all images. Maybe there is no need for it, but I think there is no need to use distinct tags either.

Reply:
it can't be the same for all of them.  they're generated in different repos in different pipelines (which is how the tag is generated)

Reply:
But we can at least reduce the number of builders variables to the number of builder repositories we have, which is 2 (instead of the 4 we have right now).